### DISCUSSION TITLE
Browse by genre

### DISCUSSION THREAD

**Discussion by XendoxShade23** [Kareadita/Kavita#3173] [Ideas]
### Idea Description

It would be nice if users could browse comics by genre without having to specifically search for one. Currently the only way to pull up a genre list is in the admin settings -> statistics, then clicking on genres. This isn't available to non admin users.

### Idea Category

User Experience

### Duration of Using Kavita

6 months.

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by majora2007** [comment]
This will be delivered with v0.8.7. In addition to this, I also brought browse by tags.

**Reply by XendoxShade23** [reply]
Awesome! Thank you so much!
